backend/src/db/db_base.py

```diff
@@ -14,6 +15,8 @@ def __init__(self, **cfg) -> None:
         self._cfg = cfg
         self._connections = set()
 
+    sqlFactory = SQLFactory
```

Comment:
it's probably clearer to implement `sqlFactory` as property to overwritten by the derived class.